extractor.go

```diff
@@ -259,17 +190,20 @@ func indexTX(out io.Writer, sync *sync.Mutex, result *abci.TxResult) error {
 
 	sync.Lock()
 	defer sync.Unlock()
-	_, err = io.WriteString(out, fmt.Sprintf("%s %d %d %s\n",
+
+	return out.WriteLine(fmt.Sprintf("%s %d %d %s",
```

Comment:
Totally unrelated to the PR itself, but I was wondering because probably we might be able to skip base64 here. But, I'm not sure how would that work with  \n separator. This should probably make this whole thing faster. Maybe it would be good idea to make it configurable instead (like utf8 0x1D character or something ).

0x1D
--

Reply:
We'd still want to have base64-based (or similar) content if we're logging into STDOUT/STDERR. Maybe use a fast implementation of base58? Dunno, def worth researching once we get all our data extraction finalized.

Reply:
If we had some sort of separator we don't need to have new line always, we can do just fine using other separator, as long as it won't happen inside any raw data

Reply:
Cool, i'll add a ticket for this